### PR TITLE
add tasks for portals and authentication

### PR DESCRIPTION
**Pull request by SlothOfAnarchy**

### CODE REVIEW

File: tasks/configure-target.yml [hunk 1]
```diff
@@ -24,7 +12,27 @@
...
+    backstore_name: "{{ disk.name }}"
+    options: " {{ iscsi_disk_path_prefix | default('') }}{{ disk.path }} "
+  with_items:
+    - "{{ target.disks }}"
+  loop_control:
+    loop_var: "disk"
+  notify:
+    - save targetcli configuration
+
+- name: create portal for target {{ target.name }}
```
Comment (dasJ) [inline tasks/configure-target.yml]: Capitalize

File: tasks/configure-target.yml [hunk 2]
```diff
@@ -24,7 +12,27 @@
   with_items:
     - "{{ target.initiators }}"
   loop_control:
-    loop_var: initiator
+    loop_var: "initiator"
+  notify:
+    - save targetcli configuration
+
+- name: define backstore objects for target {{ target.name }}
```
Comment (dasJ) [inline tasks/configure-target.yml]: Capitalize

File: tasks/configure-target.yml [hunk 3]
```diff
@@ -36,6 +44,16 @@
   with_items:
     - "{{ target.disks }}"
   loop_control:
-    loop_var: disk
+    loop_var: "disk"
+  notify:
+    - save targetcli configuration
+
+- name: configure authentication for target {{ target.name }}
```
Comment (dasJ) [inline tasks/configure-target.yml]: Capitalize

File: README.md [hunk 4]
```diff
@@ -9,29 +9,53 @@ Also, this role is not creating any disks/partitions/LVs. Therefore it is expect
...
-| Name                  | Description                                                                                 |
-|-----------------------|---------------------------------------------------------------------------------------------|
-| `name`                | `name` identifier of this target. This is appended to `iscsi_base_wwn` and used as `wwn`    |
-| `disks`               | disk configuration, see [below](#disks)                                                     |
-| `initiators`          | list of `initiator`-`wwn`s                                                                  |
 
-Optionally, a `iscsi_disk_path_prefix` var can be added at the top level and will be prepended before each disk `path`.
+### targets
+
+each target has the following mandatory vars:
```
Comment (dasJ) [inline README.md]: Capitalize

File: README.md [hunk 5]
```diff
@@ -9,29 +9,53 @@ Also, this role is not creating any disks/partitions/LVs. Therefore it is expect
...
+
+| Name                  | mandatory | Description                                                                            |
+|-----------------------|--------|-----------------------------------------------------------------------------------------|
+| `name`                | yes |`name` identifier of this target. This is appended to `iscsi_base_wwn` and used as `wwn`    |
```
Comment (dasJ) [inline README.md]: Capitalize each description

File: README.md [hunk 6]
```diff
@@ -9,29 +9,53 @@ Also, this role is not creating any disks/partitions/LVs. Therefore it is expect
...
-| `name`                | `name` identifier of this target. This is appended to `iscsi_base_wwn` and used as `wwn`    |
-| `disks`               | disk configuration, see [below](#disks)                                                     |
-| `initiators`          | list of `initiator`-`wwn`s                                                                  |
 
-Optionally, a `iscsi_disk_path_prefix` var can be added at the top level and will be prepended before each disk `path`.
+### targets
+
+each target has the following mandatory vars:
+
+| Name                  | mandatory | Description                                                                            |
```
Comment (dasJ) [inline README.md]: Capitalize `mandatory`

File: README.md [hunk 7]
```diff
@@ -9,29 +9,53 @@ Also, this role is not creating any disks/partitions/LVs. Therefore it is expect
...
+| `path`                | existing path to the disk that should be used as backstore. `iscsi_disk_path_prefix` will be prepended if defined. |
 | `type`                | one out of {`fileio`, `iblock`, `pscsi`, `rd_mcp`}                                                      |
 
+### authentication
+
+`authentication` is a dict containing the following entries:
+
+| Name                  | mandatory  | Description                                                                                  |
+|-----------------------|------------|---------------------------------------------------------------------------------------------|
+| `userid`              | yes  | userid to authenticate the initator |
```
Comment (dasJ) [inline README.md]: Fix columns and capitalize

Comment (SlothOfAnarchy) [inline README.md]: why? The table is shown as intended, that's what counts.

**Comment by SlothOfAnarchy** [comment]
@dasJ please rereview

**Pull request merged**